met-etl/migrations/versions/2c7d5ce6e117_analytics_db.py

```diff
@@ -0,0 +1,210 @@
+"""analytics_db
+
+Revision ID: 2c7d5ce6e117
+Revises: 
+Create Date: 2022-07-22 16:17:22.886477
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2c7d5ce6e117'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('engagement',
+    sa.Column('engagement_id', sa.Integer(), nullable=False),
+    sa.Column('engagement_name', sa.String(length=50), nullable=True),
+    sa.Column('start_date', sa.DateTime(), nullable=True),
+    sa.Column('end_date', sa.DateTime(), nullable=True),
+    sa.Column('status_name', sa.String(length=50), nullable=True),
+    sa.Column('created_date', sa.DateTime(), nullable=True),
+    sa.Column('updated_date', sa.DateTime(), nullable=True),
+    sa.Column('published_date', sa.DateTime(), nullable=True),
+    sa.PrimaryKeyConstraint('engagement_id')
+    )
+    op.create_table('user_table',
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('user_name', sa.String(length=100), nullable=True),
+    sa.Column('created_date', sa.DateTime(), nullable=True),
+    sa.Column('updated_date', sa.DateTime(), nullable=True),
+    sa.PrimaryKeyConstraint('user_id')
+    )
+    op.create_table('survey',
+    sa.Column('survey_id', sa.Integer(), nullable=False),
+    sa.Column('survey_name', sa.String(length=50), nullable=True),
+    sa.Column('engagement_id', sa.Integer(), nullable=True),
+    sa.Column('created_date', sa.DateTime(), nullable=True),
+    sa.Column('updated_date', sa.DateTime(), nullable=True),
+    sa.Column('version', sa.String(length=50), nullable=True),
+    sa.Column('active_flag', sa.String(length=1), nullable=True),
+    sa.ForeignKeyConstraint(['engagement_id'], ['engagement.engagement_id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('survey_id')
+    )
+    op.create_table('request_type_radio',
+    sa.Column('request_id', sa.Integer(), nullable=False),
+    sa.Column('survey_id', sa.Integer(), nullable=False),
+    sa.Column('engagement_id', sa.Integer(), nullable=False),
+    sa.Column('request_key', sa.String(length=100), nullable=False),
+    sa.Column('request_type', sa.String(length=100), nullable=True),
+    sa.Column('request_label', sa.String(length=100), nullable=True),
```

Comment:
same here..does it make sense to remove request from column names?

Reply:
Addressed